### DISCUSSION TITLE
When is the Spring  6 support scheduled?

### DISCUSSION THREAD

**Discussion by RezaLabudeBSM** [jdbi/jdbi#2613] [Ideas]
Hello,

i would love to use JDBI, but I'm "stuck" with the most actual Spring 6 Stack and as far as i saw JDBI is still on version 5. So my question is when are you planning to release a spring 6 version of JDBI? Is there a place to look up a road-map?
best regards

Reza

**Comment by stevenschlansker** [comment]
I am not sure if anyone has tried using Spring 6 yet. If you try to use the existing Spring 5 support, it might just work. If not, please file an issue with the failure you see and hopefully we can improve compatibility.

**Comment by RezaLabudeBSM** [comment]
I did a first test and it looks very good in my sample project with a spring 6.1.2 environment. I was also able to run a mvn clean install in the jdbi project for spring5 after changing the spring version to 6.1.2 in the parent bom. So i would guess it runs out of the box as they did not change anything important for jdbi in the relevant spring projects. Can you therefore please officially release spring 6 in the short term future?
best

**Reply by stevenschlansker** [reply]
Hi @RezaLabudeBSM , thanks for your interest. Jdbi is a community project and there is no 'official support' channel: users will have to collaborate among themselves and with the developers to build support for new features.

Thank you for testing Jdbi against Spring 6. While we can't promise that there will be no problems, I have updated the documentation to show that Spring 6 should work with Jdbi. We won't add a specific "spring6" module until we need to use new features introduced only in Spring 6. Until then, "spring5" will remain the correct artifact to use even for Spring 6 support.

https://github.com/jdbi/jdbi/pull/2620

If you notice any missing features or bugs, please report them :)

**Reply by stevenschlansker** [reply]
I updated the documentation to explicitly list Spring 6 support.
